### ISSUE TITLE
Add WP and PHP requirements policy

### ISSUE THREAD

**Issue by kasparsd** [WordPress/two-factor#650]
### Is your enhancement related to a problem? Please describe.

Considering the important use-case of this plugin, users should be able to plan for keeping their infrastructure up to date with the requirements of this plugin. 

### Proposed Solution

Add a section to the readme.txt that documents the update policy for WP core and PHP requirements.

See #641 for relevant discussion.

```
= PHP and WordPress Version Support =

This plugin supports the last two major versions of WordPress and <a href="https://make.wordpress.org/core/handbook/references/php-compatibility-and-wordpress-versions/">the minimum PHP version</a> supported by those WordPress versions.
```

### Designs

No impact.

### Please confirm that you have searched existing issues in this repository.

Yes

**Comment by kasparsd** [comment]
@jeffpaul Can you please suggest the exact approach and wording here?

**Comment by jeffpaul** [comment]
Perhaps something in the `== Frequently Asked Questions ==` section like:

```
= What PHP and WordPress versions does the Two-Factor plugin support? =

This plugin supports the last two major versions of WordPress and <a href="https://make.wordpress.org/core/handbook/references/php-compatibility-and-wordpress-versions/">the minimum PHP version</a> supported by those WordPress versions.
```

As of today, that would allow us to bump the WordPress minimum to 6.7:
https://github.com/WordPress/two-factor/blob/fe16f23875a5aeeb4d76f24f4b501674460f6a07/two-factor.php#L15
https://github.com/WordPress/two-factor/blob/fe16f23875a5aeeb4d76f24f4b501674460f6a07/.github/workflows/test.yml#L82

and bump the PHP minimum to 7.2:
https://github.com/WordPress/two-factor/blob/fe16f23875a5aeeb4d76f24f4b501674460f6a07/two-factor.php#L16
https://github.com/WordPress/two-factor/blob/fe16f23875a5aeeb4d76f24f4b501674460f6a07/composer.json#L15
https://github.com/WordPress/two-factor/blob/fe16f23875a5aeeb4d76f24f4b501674460f6a07/composer.json#L24
https://github.com/WordPress/two-factor/blob/fe16f23875a5aeeb4d76f24f4b501674460f6a07/.github/workflows/test.yml#L70

**Comment by kasparsd** [comment]
Yes, that would be great! Can you open a pull request with that or would you prefer that somebody else works on that @jeffpaul?
